ooni/nettests/blocking/telegram.py

```diff
@@ -0,0 +1,90 @@
+# -*- encoding: utf-8 -*-
+
+from twisted.internet import defer, reactor
+from twisted.python import usage
+from twisted.internet.endpoints import TCP4ClientEndpoint
+
+from ooni.utils import log
+from ooni.common.tcp_utils import TCPConnectFactory
+from ooni.errors import failureToString
+
+from ooni import nettest
+
+# These are taken from:
+# https://github.com/telegramdesktop/tdesktop/blob/e6d94b5ee7d96a97ee5976dacb87bafd00beac1d/Telegram/SourceFiles/config.h#L205
+TELEGRAM_DCS = [
+    (1, "149.154.175.50", 443),
+    (2, "149.154.167.51", 443),
+    (3, "149.154.175.100", 443),
+    (4, "149.154.167.91", 443),
+    (5, "149.154.171.5", 443)
```

Comment:
None of those are official telegram implementation, so I am not going to include it.

In looking at existing telegram implementations there is a pretty big variance in the servers they connect to and looking at their history I see them changing them from time to time, because telegram stops using them.

We want the tested endpoints to be reliable for testing and not have to update them often.